.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+
+test-defaults: &test-defaults
+  working_directory: ~/graphql-directive-uid
+  steps:
+    - attach_workspace:
+        at: ~/graphql-directive-uid
+    - run: |
+        yarn lint
+        yarn test
+
+jobs:
+  install-dependencies:
+    working_directory: ~/graphql-directive-uid
+    docker:
+      - image: circleci/node:6
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/graphql-directive-uid
+      - restore_cache:
+          keys:
+            - dependencies-{{ checksum "package.json" }}
```

Comment:
Calculate checksum from lockfile? Cache will be invalidated after adding npm-scripts or such